### PR TITLE
[IMP] base: default name for ir.actions.act_window

### PR DESCRIPTION
**Pull request by rco-odoo**
Take the name of the model if no name is set for an action.

### CODE REVIEW

File: odoo/addons/base/models/ir_actions.py [hunk 1]
```diff
@@ -234,6 +234,9 @@ def for_xml_id(self, module, xml_id):
     @api.model_create_multi
     def create(self, vals_list):
         self.clear_caches()
+        for vals in vals_list:
+            if not vals.get('name'):
```
Comment (odony) [inline odoo/addons/base/models/ir_actions.py]:
```suggestion
            if not vals.get('name') and vals.get('res_model'):
```

**Comment by rco-odoo** [comment]
@pedrobaeza the main use case for this is to omit the action name in the XML data of a module. In that case, no translation is needed.

**Comment by pedrobaeza** [comment]
But the action name is used on the view title, so translation can be important, isn't it?

**Comment by rco-odoo** [comment]
@pedrobaeza in case this is used in a module's XML files, the value will end up in the translation files (the `pot` files are generated from what the database contains after having installed the modules), and will coincide with the model's description in `ir.model`.

**Comment by pedrobaeza** [comment]
OK, thanks for the extra explanation. Now I get it.

**Comment by rco-odoo** [comment]
robodoo r+

**Comment by rco-odoo** [comment]
@robodoo retry

**Pull request closed without merging**